fe/fe-core/src/main/java/com/starrocks/credential/aws/AWSCloudConfiguration.java

```diff
@@ -50,6 +50,15 @@ public AWSCloudCredential getAWSCloudCredential() {
 
     @Override
     public void applyToConfiguration(Configuration configuration) {
+        configuration.set("fs.s3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
+        configuration.set("fs.s3a.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
+        // Below storage using s3 compatible storage api
+        configuration.set("fs.oss.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
+        configuration.set("fs.ks3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
```

Comment:
Is it better to define some constants?

Reply:
I think there is no need to define constants because there are too many Hadoop configuration key here, we don't need to define a constant for each key.